### PR TITLE
Disable Google Optimize script

### PR DESCRIPTION
**Pull request by barrymcgee**
## Done

Disable Google Optimize script as it's not being used.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)

### CODE REVIEW

File: templates/templates/_tag_manager.html [hunk 1]
```diff
@@ -1,10 +1,10 @@
         <!-- Google Optimize page-hiding -->
-        <style>.async-hide { opacity: 0 !important} </style>
+        <!-- <style>.async-hide { opacity: 0 !important} </style>
```
Comment (pmahnke) [inline templates/templates/_tag_manager.html]:
```suggestion
        <style>.async-hide { opacity: 0 !important} </style>
```

File: templates/templates/_tag_manager.html [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
         <script>(function(a,s,y,n,c,h,i,d,e){s.className+=' '+y;h.start=1*new Date;
         h.end=i=function(){s.className=s.className.replace(RegExp(' ?'+y),'')};
         (a[n]=a[n]||[]).hide=h;setTimeout(function(){i();h.end=null},c);h.timeout=c;
         })(window,document.documentElement,'async-hide','dataLayer',1000,
-        {'GTM-N2MDH37':true});</script>
+        {'GTM-N2MDH37':true});</script> -->
```
Comment (pmahnke) [inline templates/templates/_tag_manager.html]:
```suggestion
        {'GTM-N2MDH37':true});</script>
{% endcomment %}
```

File: templates/templates/_tag_manager.html [hunk 3]
```diff
@@ -1,10 +1,10 @@
         <!-- Google Optimize page-hiding -->
```
Comment (pmahnke) [inline templates/templates/_tag_manager.html]:
```suggestion
{% comment %}
        <!-- Google Optimize page-hiding -->
```

**Comment by nottrobin** [comment]
So unfortunately we do still get a flash of the default font, which looks pretty jarring:

image

I think we need to make a call right now about whether this unpleasant experience for people's first visit more, or the ~0.5s delay in showing the page more.

Either way, I think we should find a way to inline these fonts in our CSS as data-uris, possibly using some of @hatched's pruning tricks.

**Comment by barrymcgee** [comment]
@nottrobin Added your FOUT issue here: https://github.com/canonical-websites/www.ubuntu.com/issues/4756

**Comment by nottrobin** [comment]
Thanks @barrymcgee. I'm happy to merge this without fixing the FOUT issue, but it would be good if we could focus on the issue as a priority.

Having [discovered](https://github.com/canonical-websites/www.ubuntu.com/issues/4756#issuecomment-467406286) that the preconnect stuff seemingly doesn't have any impact at all, I really do think the only actual solution is to inline our fonts inside the CSS, however we do that.

**Pull request merged**